.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+  schedule:
+    # Run a cron job once daily
+    - cron: "0 0 * * *"
```

Comment:
Does this mean that GHA will run `pytest` once a day?

Reply:
Exactly. The main value here, in my experience, is detecting upstream changes and API breaks here, instead of while you're working on a PR or doing science.

Travis had this feature but you had to enable it through the website UI, so I don't think many people used it.